src/common/circle-series.component.ts

```diff
@@ -44,7 +44,7 @@ import { id } from '../utils/id';
         ngx-tooltip
         [tooltipPlacement]="'top'"
         [tooltipType]="'tooltip'"
-        [tooltipTitle]="getTooltipText(circle)"
+        [tooltipTitle]="getTooltipText(circle, circle.min, circle.max)"
```

Comment:
min and max are properties on circle, so there's no need to be passed separately here.